app/views/overrides/activation_keys/_host_environment_select.html.erb

```diff
@@ -15,26 +15,33 @@
 <% cs_select_id =  using_hostgroups_page? ? :content_source_id : :content_source_id %>
 <% cs_select_name =  using_hostgroups_page? ? 'hostgroup[content_source_id]' : 'host[content_facet_attributes][content_source_id]' %>
 <% cs_select_attr = using_hostgroups_page? ? 'content_source' : 'content_facet.content_source' %>
+<% content_source_inherited = true if content_source_inherited? %>
 
-<%= field(f, cs_select_attr, {:label => _("Content Source")}) do
-  if using_hostgroups_page?
-    select_tag cs_select_id, content_source_options(@hostgroup, :include_blank => blank_or_inherit_with_id(f, :content_source)), :data => {"spinner_path" => spinner_path},
-               :class => 'form-control',  :name => cs_select_name
-  else
-    select_tag cs_select_id, content_source_options(@host, :selected_host_group => @hostgroup || @host.hostgroup, :include_blank => blank_or_inherit_with_id(f, :content_source)), :data => {"spinner_path" => spinner_path}, :class => 'form-control',  :name => cs_select_name, :disabled => cv_lce_disabled?
-  end
-end %>
+<%= field(f, cs_select_attr, {:label => _("Content Source"), :help_inline => content_source_inherited ? 'Inherited' : nil }) do %>
```

Comment:
You can just call the methods directly here like this, no? And then you can eliminate the four variable assignments.

```suggestion
<%= field(f, cs_select_attr, {:label => _("Content Source"), :help_inline => content_source_inherited? ? 'Inherited' : nil }) do %>
```